plugins/jenkins-backend/api-report.md

```diff
@@ -28,6 +28,7 @@ export class DefaultJenkinsInfoProvider implements JenkinsInfoProvider {
   getInstance(opt: {
     entityRef: CompoundEntityRef;
     jobFullName?: string;
+    token?: string;
```

Comment:
I wonder if `backstageToken` is a better name here rather than just token.

Reply:
The ‘token’ name is used widely in backstage therefore I prefer to use this name, but I you think it can be better for this plugin we can rename it :)

Reply:
In this case however, we don't want to get it confused with token used for jenkins, so I think it makes sense to have it as `backstageToken` at least in the interface for jenkins.

Reply:
@benjdlambert  I've renamed token to backstageToken

Reply:
@benjdlambert  can we merge this change today? I think today backstage will release a new version so we can take this one too :)

Reply:
The releases are on Tuesdays now.

Reply:
@benjdlambert  @freben thanks for your feedback and approval! :)